src/subscription_manager/repofile.py

```diff
@@ -72,6 +72,9 @@ class Repo(dict):
             'ui_repoid_vars': (0, None)}
 
     def __init__(self, repo_id, existing_values=None):
+        if HAS_DEB822 is True:
+            self.PROPERTIES['arches'] = (1, '0')
```

Comment:
`arches` is a list of values and not a boolean, so I'd use `None` as default value here instead of `0`